server/src/main/java/org/apache/kafka/security/authorizer/AclEntry.java

```diff
@@ -90,63 +65,6 @@ public AclEntry(AccessControlEntry ace) {
             : SecurityUtils.parseKafkaPrincipal(ace.principal());
```

Comment:
Is the constructor of `AclEntry` used in production code? if not, could you please remove it in this PR?